web/src/common/utils.ts

```diff
@@ -115,3 +117,28 @@ export function dateTimeLocal(date: Date): string {
     const parts = localISOTime.split(":");
     return `${parts[0]}:${parts[1]}`;
 }
+
+// Lit is extremely well-typed with regard to CSS, and Storybook's `build` does not currently have a
+// coherent way of importing CSS-as-text into CSSStyleSheet. It works well when Storybook is running
+// in `dev,` but in `build` it fails. Storied components will have to map their textual CSS imports
+// using the function below.
+type AdaptableStylesheet = Readonly<string | CSSResult | CSSStyleSheet>;
+type AdaptedStylesheets = CSSStyleSheet | CSSStyleSheet[];
+
+const isCSSResult = (v: any): v is CSSResult =>
+    v instanceof CSSResult && v.styleSheet !== undefined;
+
+// prettier-ignore
+export const _adaptCSS = (sheet: AdaptableStylesheet): CSSStyleSheet =>
+    (typeof sheet === "string" ? css([sheet] as unknown as TemplateStringsArray, ...[]).styleSheet
```

Comment:
Very small nitpick, I think this makes more sense as a full function instead of a tertiary statement